storage/innobase/include/os0file.h

```diff
@@ -470,9 +470,12 @@ struct Encryption {
   @param[in]      algorithm       Encryption algorithm to check
```

Comment:
missing doxygen comment for explicit_encryption parameter

Reply:
done